README.md

```diff
@@ -0,0 +1,109 @@
+[![Build Status](https://travis-ci.org/alper-oner/dasher.svg?branch=master)](https://travis-ci.org/alper-oner/dasher)
```

Comment:
Please leave the README file as is. So do not delete it. 

You may want to pull the repository because there were updated and you may get conflicts now.

You will get a README.md file when you pull and then you can add your Build status with prefix "Java:" to it.